torch/csrc/jit/passes/onnx.cpp

```diff
@@ -343,16 +343,17 @@ void NodeToONNX(
     if (func) {
       pyobj = func->get();
     }
-    if (!py::hasattr(pyobj, "symbolic")) {
+
+    py::object is_registered_op = onnx_registry.attr("is_registered_op")("prim_PythonOp", "", 9);
+    if (!py::hasattr(pyobj, "symbolic") && (!PyObject_IsTrue(is_registered_op.ptr()))) {
```

Comment:
nit: It would be good to put a comment with a brief explanation (especially about 'symbolic') here.